### ISSUE TITLE
One test is failing now: about Gaussian fitting

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#113]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/113) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Not sure why this started, but we are now failing a math (Gaussian fitting) test. Possibly the random seed has changed?

```text
======================================================================
FAIL: test_penalty_haseffect (test_fitting.Test_gaussian)
Test the regularization penalty in case where it has an effect
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/fowlerj/Software/mass/src/mass/mathstat/test/test_fitting.py", line 176, in test_penalty_haseffect
    params = self.run_several_fits(1000, nfits, fwhm, ctr, nbins, N_bg, penalty=penalty)
  File "/Users/fowlerj/Software/mass/src/mass/mathstat/test/test_fitting.py", line 126, in run_several_fits
    self.assertTrue(np.all(sigma_errors[:maxparam].std(axis=1) < 1+2/nfits**0.5))
AssertionError: False is not true

----------------------------------------------------------------------
```

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Use fixed random seeds to prevent test of Gaussian_fitter failing.

I thought I fixed this in 42c3d77 but of course the fix only lasted
until new tests changed the random seed. You have to fix the seed 
specifically for the test class that’s failing. Fixes #113.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.6 (automated comment)
